package.json

```
@@ -20,7 +20,7 @@
     "check:test": "run-s test -- --silent",
     "check:types": "tsc --build",
     "chore:format:fix": "prettier --cache --write .",
-    "chore:lint:fix": "pnpm check:lint -- --quiet --fix",
+    "chore:lint:fix": "run-s check:lint -- --quiet --fix",
```

might be better to just use turbo directly here I think the args passing multiple times gets messed up

```suggestion
    "chore:lint:fix": "turbo run lint -- --quiet --fix",
```

---

Done. Thanks!